src/components/diagnosticExercise/config/route.js

```diff
@@ -0,0 +1,144 @@
+(function (angular) {
+    'use strict';
+
+    angular.module('znk.infra-web-app.diagnosticExercise').config([
+        '$stateProvider',
+        function ($stateProvider) {
+            $stateProvider
+                .state('diagnostic', {
+                    url: '/diagnostic?skipIntro',
+                    templateUrl: 'components/diagnosticExercise/templates/workoutsDiagnostic.template.html',
+                    resolve: {
+                        currentState: function currentState(WorkoutsDiagnosticFlow, $stateParams) {
+                            'ngInject';
+                            return WorkoutsDiagnosticFlow.getDiagnosticFlowCurrentState(null, $stateParams.skipIntro);
+                        }
+                    },
+                    controller: 'WorkoutsDiagnosticController',
+                    controllerAs: 'vm'
+                })
+                .state('diagnostic.intro', {
+                    url: '/intro/:id/:subjectId/:order',
```

Comment:
so change to sectionId why do we need the order in the url ?
Why we need the url here ?